enterprise/jmx-monitoring/src/test/java/io/crate/beans/QueryStatsTest.java

```diff
@@ -76,122 +73,74 @@ private MetricsView createFailedExecutionMetric(Classification classification, l
         assertThat(metricsByCommand.get(StatementType.SELECT).sumOfDurations(), is(90L));
-        assertThat(metricsByCommand.get(StatementType.SELECT).avgDurationInMs(), is(30.0));
-        assertThat(metricsByCommand.get(StatementType.SELECT).statementsPerSec(), is(1.5));
 
         assertThat(metricsByCommand.get(StatementType.INSERT).totalCount(), is(1L));
         assertThat(metricsByCommand.get(StatementType.INSERT).failedCount(), is(0L));
         assertThat(metricsByCommand.get(StatementType.INSERT).sumOfDurations(), is(19L));
-        assertThat(metricsByCommand.get(StatementType.INSERT).avgDurationInMs(), is(19.0));
-        assertThat(metricsByCommand.get(StatementType.INSERT).statementsPerSec(), is(0.5));
 
         assertThat(metricsByCommand.get(StatementType.UPDATE).totalCount(), is(1L));
         assertThat(metricsByCommand.get(StatementType.UPDATE).failedCount(), is(0L));
         assertThat(metricsByCommand.get(StatementType.UPDATE).sumOfDurations(), is(20L));
-        assertThat(metricsByCommand.get(StatementType.UPDATE).avgDurationInMs(), is(20.0));
-        assertThat(metricsByCommand.get(StatementType.UPDATE).statementsPerSec(), is(0.5));
 
         assertThat(metricsByCommand.get(StatementType.DELETE).totalCount(), is(2L));
         assertThat(metricsByCommand.get(StatementType.DELETE).failedCount(), is(0L));
         assertThat(metricsByCommand.get(StatementType.DELETE).sumOfDurations(), is(15L));
-        assertThat(metricsByCommand.get(StatementType.DELETE).avgDurationInMs(), is(7.5));
-        assertThat(metricsByCommand.get(StatementType.DELETE).statementsPerSec(), is(1.0));
 
         assertThat(metricsByCommand.get(StatementType.DDL).totalCount(), is(1L));
         assertThat(metricsByCommand.get(StatementType.DDL).failedCount(), is(0L));
         assertThat(metricsByCommand.get(StatementType.DDL).sumOfDurations(), is(1L));
-        assertThat(metricsByCommand.get(StatementType.DDL).avgDurationInMs(), is(1.0));
-        assertThat(metricsByCommand.get(StatementType.DDL).statementsPerSec(), is(0.5));
 
         assertThat(metricsByCommand.get(StatementType.COPY).totalCount(), is(1L));
         assertThat(metricsByCommand.get(StatementType.COPY).failedCount(), is(1L));
         assertThat(metricsByCommand.get(StatementType.COPY).sumOfDurations(), is(0L));
-        assertThat(metricsByCommand.get(StatementType.COPY).avgDurationInMs(), is(0.0));
-        assertThat(metricsByCommand.get(StatementType.COPY).statementsPerSec(), is(0.5));
-
-        assertThat(metricsByCommand.get(StatementType.ALL).totalCount(), is(9L));
-        assertThat(metricsByCommand.get(StatementType.ALL).sumOfDurations(), is(145L));
-        assertThat(metricsByCommand.get(StatementType.ALL).avgDurationInMs(), is(16.11111111111111));
-        assertThat(metricsByCommand.get(StatementType.ALL).statementsPerSec(), is(4.5));
     }
 
     @Test
     public void testAveragesSinceLastReadAndGlobalSumOfDurationsAndCount() {
```

Comment:
Maybe adjust the test name as well?
Or drop it all together - looks very similar to the one above if not the same